packages/lib-classifier/src/plugins/tasks/experimental/volumetric/models/VolumetricTask.js

```diff
@@ -0,0 +1,23 @@
+import cuid from "cuid"
```

Comment:
A comment just for awareness that if you'e going to continue using `cuid` it is a deprecated third-party package as documented in https://github.com/zooniverse/front-end-monorepo/issues/6493. All of the other task models still use it so this is just an fyi.

Reply:
Yup - thank you for the cross-reference visibility